### ISSUE TITLE
Broken inheritance

### ISSUE THREAD

**Issue by nippur72** [nippur72/PolymerTS#18]
It's not possible to have an element-class that extends another class that is not `polymer.Base`:
```TypeScript
class MyAbstract extends polymer.Base implements polymer.Element {
  // methods here are not carried in the final polymer element
}

class My extends MyAbstract  {
  private myProp1: string = 'initialVal';
}
```

**Comment by nippur72** [comment]
fixed with #23

**Comment by lgrignon** [comment]
Hello, it's me again :)

I tried the brand new 0.1.15 version of polymer-ts and it seems that inheritance is broken again :(

```typescript
class BaseComponent extends polymer.Base implements polymer.Element {
   // methods here won't be available at runtie
}

class SomeAbstractComponent extends BaseComponent {
}

@component('my-comp')
class MyComp extends SomeAbstractComponent {
}
```

Any idea on this?

**Comment by nippur72** [comment]
I'm looking at the tests right now. 

BTW `implements polymer.Element` is no longer necessary :-)

**Comment by nippur72** [comment]
Well, one of the test was ill, or perhaps I made it that way and I don't remember.

Since we switched to the newest (instance-less) initialization method, PolymerTS is no longer able to take inherited methods to the polymer elements. This because such methods are in `element.prototype.__proto__` and not in just `element.prototype`.

Can't think of a quick fix for this, perhaps we should manually (and recursively) copy from `__proto__`.
